spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -86,6 +86,7 @@ def get_parser():
         choices=[0, 1],
         default=0,
         help="Extrapolate beyond the segmentation boundaries. 0 = no extrapolation, 1 = extrapolation. Only relevant with -method fitseg."
+             "Note: '-extrapolation 1' works best with lower-order centerline fitting algorithms"
```

Comment:
Tiny tiny nitpick: It might be good to add in brackets which methods are lower-order? (Is it just linear? Or linear and nurbs?)

(I mention this because right now, the `-centerline-algo` flag doesn't contain any info on the various algorithms, so the user might not know what "lower-order" means.)

Reply:
nurbs too! done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4079/commits/483e72f0d9a8edc933b1629410c7eae5e81a83f0